src/main.cpp

```diff
@@ -164,6 +164,7 @@ static void lPrintVersion() {
     printf("    [--werror]\t\t\t\tTreat warnings as errors\n");
     printf("    [--woff]\t\t\t\tDisable warnings\n");
     printf("    [--wno-perf]\t\t\tDon't issue warnings related to performance-related issues\n");
+    printf("    [--[no-]wrap-signed-int]\t\tDo not preserve/Preserve wrap-around on signed integer overflow\n");
```

Comment:
```suggestion
    printf("    [--[no-]wrap-signed-int]\t\t[Do not] preserve wraparound on signed integer overflow\n");
```

- need to clearly indicate what that [no] corresponds to [do not]
- "wraparound" is used by clang community instead of "wrap-around", while both version are legal English words

Reply:
Fixed.